tests/runtime/call

```diff
@@ -92,6 +92,16 @@ EXPECT @bytes: *\n[\[(].*
 TIMEOUT 5
 AFTER cat /dev/null
 
+NAME lhist_negative_arg
+RUN bpftrace -v -e 'kretprobe:vfs_read { @bytes = lhist(retval, -10, 10, 1); exit()}'
+EXPECT lhist\(\) expects a integer literal \(integer provided\)
+TIMEOUT 5
+
+NAME lhist_nonliteral_arg
+RUN bpftrace -v -e 'kretprobe:vfs_read { $s = "asdf"; @bytes = lhist(retval, 0, $s, 1); exit()}'
```

Comment:
We typically do this kind of thing in unit tests in `tests/semantic_analyser.cpp`. No need to run the heavyweight tests for checking invalid programs

Reply:
We've already got one for `hist()`, but `lhist()` is missing: https://github.com/iovisor/bpftrace/blob/a4bf870ea9733cf7ef048cca367dc3042b7a0cef/tests/semantic_analyser.cpp#L155